src/components/organisms/CreateAccount.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react';
```

Comment:
**FOR THIS ENTIRE FILE**
There'll be one component called Authenticate at the route _/authenticate_. You can rename the login component and route to authenticate. In this component, there'll be a ternary that shows either login or signup input fields. We don't need to have separate routes and extra code for something this simple